src/main/io/serial.c

```diff
@@ -476,7 +479,7 @@ void serialInit(bool softserialEnabled, serialPortIdentifier_e serialPortToDisab
         }
 
 #if !defined(SIMULATOR_BUILD)
-        else if (serialPortUsageList[index].identifier <= SERIAL_PORT_USART8) {
+        else if (serialPortUsageList[index].identifier <= SERIAL_PORT_USART10) {
```

Comment:
Can we add `serialPortUsageList[index].identifier == SERIAL_PORT_LPUART1` here? Maybe with ifdef or so?

It seems to be duplicated code on line 505 and below.

Reply:
Sure, was already about doing that, but softserial was in between and LPUART1 was an addition.

Reply:
Fixed.

Reply:
LGTM